examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>Can you add a bit of text what the user is seeing in the image?  Otherwise hard to understand</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>